src/bridge/nid_util/nid_util_test.cc

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+#include <openssl/evp.h>
+
+#include "src/backing/client/digest_case.h"
+#include "src/bridge/nid_util/nid_util.h"
+#include "src/testing_util/test_matchers.h"
+
+namespace kmsengine {
+namespace bridge {
+namespace {
+
+using ::kmsengine::backing::DigestCase;
+using ::kmsengine::testing_util::IsOk;
+using ::testing::Not;
+using ::testing::Values;
+using ::testing::ValuesIn;
+
+// Mapping from OpenSSL NIDs associated with `EVP_PKEY` objects to their
+// engine-native equivalent.
+struct CorrespondingNid {
+  int actual;
+  DigestCase expected;
+} const kNidMapping[]{
```

Comment:
Nit: separate type definition from variable declaration as discussed previously

Reply:
Handling in #77.